trj_comau:
  joint_names: [comau_joint_1, comau_joint_2, comau_joint_3, comau_joint_4, comau_joint_5,
    comau_joint_6]
  time_from_start: [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2,
    1.3, 1.4, 1.441459992]
  positions:
  - [-0.2112901716661058, 4.578093085438012e-05, -2.255239459572047, 9.99764123465866e-05,
    -9.888182380236686e-05, 8.620494394563138e-05]
  - [-0.2162901716661058, 4.658360322440346e-05, -2.2574328541804705, 9.991653209839136e-05,
    -9.910530345365518e-05, 8.636018422885086e-05]
  - [-0.2312901716661058, 4.8991620334473445e-05, -2.264013038005741, 9.973689135380563e-05,
    -9.977574240752009e-05, 8.682590507850927e-05]
  - [-0.2562901716661058, 5.300498218459011e-05, -2.274980011047858, 9.943749011282941e-05,
    -0.00010089314066396162, 8.760210649460664e-05]
  - [-0.2912901716661056, 5.86236887747534e-05, -2.2903337733068216, 9.90183283754627e-05,
    -0.00010245749822297977, 8.868878847714294e-05]
  - [-0.3362901716661054, 6.584774010496335e-05, -2.3100743247826325, 9.847940614170552e-05,
    -0.00010446881508457452, 9.00859510261182e-05]
  - [-0.3912901691735285, 7.46771357750755e-05, -2.3342016643818493, 9.782072344140905e-05,
    -0.00010692709113733784, 9.179359406414273e-05]
  - [-0.45629014798394774, 8.51118731837206e-05, -2.3627157849959315, 9.704228046863837e-05,
    -0.00010983232565699782, 9.381171708810058e-05]
  - [-0.5250064100309116, 9.614320216699681e-05, -2.392860160732896, 9.621933110335405e-05,
    -0.00011290366291301609, 9.594522348449141e-05]
  - [-0.584152409282203, 0.00010563817404628327, -2.418806263906416, 9.551099568036945e-05,
    -0.00011554724837057153, 9.778159181950524e-05]
  - [-0.6332984085334941, 0.000113527801185523, -2.440365577863089, 9.492242075377534e-05,
    -0.00011774387452555031, 9.930747958808015e-05]
  - [-0.6724444077847851, 0.00011981208358471606, -2.457538102602914, 9.445360632357171e-05,
    -0.00011949354137795248, 0.00010052288679021607]
  - [-0.7015904070360762, 0.0001244910212438625, -2.470323838125893, 9.410455238975857e-05,
    -0.00012079624892777805, 0.00010142781342591308]
  - [-0.7207364062873676, 0.00012756461416296228, -2.478722784432026, 9.387525895233591e-05,
    -0.000121651997175027, 0.00010202225949517114]
  - [-0.7298824055386588, 0.0001290328623420154, -2.482734941521311, 9.376572601130375e-05,
    -0.00012206078611969933, 0.00010230622499799024]
  - [-0.7307418710282444, 0.00012917083618231118, -2.4831119709155076, 9.375543300993742e-05,
    -0.00012209920072928072, 0.00010233290973119437]
  velocities:
  - [-0.0, 0.0, -0.0, -0.0, -0.0, 0.0]
  - [-0.0999999999999998, 1.6053447400466613e-05, -0.04386789216846849, -1.1976049639048706e-06,
    -4.46959302576613e-06, 3.104805664389457e-06]
  - [-0.19999999999999918, 3.210689480093316e-05, -0.0877357843369368, -2.395209927809736e-06,
    -8.939186051532242e-06, 6.2096113287789e-06]
  - [-0.30000000000000165, 4.8160342201400205e-05, -0.13160367650540647, -3.592814891714639e-06,
    -1.3408779077298492e-05, 9.314416993168442e-06]
  - [-0.3999999999999923, 6.421378960186534e-05, -0.17547156867387093, -4.790419855619399e-06,
    -1.7878372103064213e-05, 1.2419222657557613e-05]
  - [-0.49999999999996325, 8.026723700232732e-05, -0.21933946084232675, -5.988024819523924e-06,
    -2.2347965128829054e-05, 1.5524028321946174e-05]
  - [-0.5999997881042001, 9.632065038621915e-05, -0.2632072600565905, -7.185627245754573e-06,
    -2.6817548683716948e-05, 1.8628827407383936e-05]
  - [-0.6999997881042141, 0.00011237409778668807, -0.30707515222506526, -8.383232209659608e-06,
    -3.128714170948372e-05, 2.173363307177383e-05]
  - [-0.641459992512939, 0.00010297644249310201, -0.2813949778194428, -7.682156711798735e-06,
    -2.867065108843833e-05, 1.991608618233391e-05]
  - [-0.5414599925129246, 8.692299509263297e-05, -0.23752708565096778, -6.484551747893738e-06,
    -2.4201058062671578e-05, 1.6811280517944043e-05]
  - [-0.4414599925128574, 7.086954769215552e-05, -0.19365919348246968, -5.286946783988059e-06,
    -1.9731465036902424e-05, 1.3706474853552491e-05]
  - [-0.3414599925129095, 5.481610029169724e-05, -0.14979130131402396, -4.08934182008381e-06,
    -1.5261872011138606e-05, 1.0601669189164647e-05]
  - [-0.24145999251290726, 3.876265289123025e-05, -0.10592340914555443, -2.8917368561789108e-06,
    -1.079227898537236e-05, 7.496863524775115e-06]
  - [-0.1414599925129042, 2.270920549076312e-05, -0.062055516977084516, -1.6941318922740017e-06,
    -6.3226859596060775e-06, 4.392057860385558e-06]
  - [-0.041459992512909785, 6.655758090297379e-06, -0.018187624808618408, -4.965269283691961e-07,
    -1.8530929338401821e-06, 1.2872521959962698e-06]
  - [-9.610976136918515e-17, 1.5428929988116144e-20, -4.2161326480806605e-17, -1.1510152729544892e-21,
    -4.295715191237585e-21, 2.9840213150216557e-21]
  accelerations:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [-1.0000000000000555, 0.0001605344740046755, -0.4386789216847079, -1.1976049639049585e-05,
    -4.469593025766374e-05, 3.104805664389671e-05]
  - [-0.9999999999987232, 0.00016053447400446303, -0.4386789216841347, -1.1976049639033688e-05,
    -4.469593025760416e-05, 3.10480566438544e-05]
  - [-1.000000000002054, 0.0001605344740050028, -0.438678921685604, -1.1976049639073407e-05,
    -4.469593025775448e-05, 3.1048056643959446e-05]
  - [-0.9999999999943934, 0.00016053447400375348, -0.438678921682211, -1.197604963898114e-05,
    -4.469593025740877e-05, 3.104805664371895e-05]
  - [-0.9999999999648614, 0.00016053447399901435, -0.4386789216692424, -1.1976049638627563e-05,
    -4.469593025608897e-05, 3.104805664280082e-05]
  - [-1.0000000000189804, 0.00016053447400770455, -0.43867892169302186, -1.1976049639275634e-05,
    -4.469593025850828e-05, 3.10480566444797e-05]
  - [-1.0000000000392262, 0.00016053447401095637, -0.43867892170186645, -1.1976049639521118e-05,
    -4.469593025941874e-05, 3.1048056645110536e-05]
  - [0.9999999999572364, -0.000160534473997802, 0.4386789216658567, 1.197604963853722e-05,
    4.4695930255748766e-05, -3.104805664256755e-05]
  - [0.9999999999773846, -0.00016053447400102232, 0.43867892167473604, 1.1976049638778102e-05,
    4.46959302566489e-05, -3.10480566431888e-05]
  - [1.0000000000968348, -0.00016053447402022145, 0.4386789217272139, 1.1976049640208627e-05,
    4.469593026199454e-05, -3.104805664690315e-05]
  - [1.0000000000032896, -0.0001605344740052135, 0.4386789216861465, 1.1976049639088677e-05,
    4.4695930257809625e-05, -3.104805664399818e-05]
  - [1.0000000000115647, -0.00016053447400652833, 0.4386789216897458, 1.1976049639187368e-05,
    4.469593025817848e-05, -3.104805664425451e-05]
  - [1.000000000015214, -0.00016053447400711157, 0.4386789216913768, 1.1976049639231243e-05,
    4.469593025834406e-05, -3.1048056644368576e-05]
  - [1.0000000000019018, -0.00016053447400497272, 0.4386789216855139, 1.1976049639071482e-05,
    4.4695930257746416e-05, -3.104805664395373e-05]
  - [1.000000000034431, -0.00016053447401019377, 0.43867892169978984, 1.1976049639461073e-05,
    4.4695930259200385e-05, -3.104805664496364e-05]


trj_kuka:
  joint_names: [structure_joint_1, kuka_joint1, kuka_joint2, kuka_joint3, kuka_joint4,
    kuka_joint5, kuka_joint6]
  time_from_start: [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2,
    1.3, 1.4, 1.5, 1.6, 1.65341421]
  positions:
  - [1.7031635288839229, -0.129161962664035, -1.6076537894015976, 1.752271896661802,
    -2.1394163277000202e-05, 0.00013296967679634692, 6.406572116538883e-05]
  - [1.698163528883923, -0.1290182890137504, -1.607376935030844, 1.7554446897564515,
    -2.1834926246751923e-05, 0.0001324036498877944, 6.421934729701518e-05]
  - [1.6831635288839228, -0.1285872680628965, -1.6065463719185833, 1.7649630690403997,
    -2.3157215156007074e-05, 0.00013070556916213678, 6.468022569189425e-05]
  - [1.658163528883923, -0.1278688998114734, -1.605162100064815, 1.780827034513647,
    -2.5361030004765664e-05, 0.0001278754346193741, 6.544835635002601e-05]
  - [1.6281635288839231, -0.12700685790976562, -1.6035009738402937, 1.7998637930815433,
    -2.8005607823275968e-05, 0.0001244792731680589, 6.637011313978414e-05]
  - [1.5981635288839233, -0.1261448160080579, -1.6018398476157722, 1.81890055164944,
    -3.0650185641786285e-05, 0.0001210831117167437, 6.729186992954226e-05]
  - [1.5681635288839244, -0.1252827741063502, -1.6001787213912504, 1.837937310217336,
    -3.32947634602965e-05, 0.00011768695026542861, 6.821362671930035e-05]
  - [1.5381635288839255, -0.12442073220464248, -1.598517595166729, 1.8569740687852316,
    -3.593934127880671e-05, 0.00011429078881411352, 6.913538350905845e-05]
  - [1.5081635288839268, -0.12355869030293476, -1.5968564689422073, 1.8760108273531275,
    -3.8583919097316923e-05, 0.00011089462736279844, 7.005714029881653e-05]
  - [1.478163528883928, -0.12269664840122704, -1.595195342717686, 1.8950475859210236,
    -4.1228496915827136e-05, 0.00010749846591148336, 7.097889708857462e-05]
  - [1.4481635288839287, -0.12183460649951933, -1.5935342164931643, 1.914084344488919,
    -4.387307473433735e-05, 0.00010410230446016827, 7.19006538783327e-05]
  - [1.4181635288839267, -0.12097256459781151, -1.5918730902686427, 1.9331211030568172,
    -4.6517652552847846e-05, 0.0001007061430088528, 7.28224106680909e-05]
  - [1.3881635288839247, -0.12011052269610371, -1.590211964044121, 1.9521578616247153,
    -4.916223037135836e-05, 9.730998155753736e-05, 7.374416745784909e-05]
  - [1.3581635288839227, -0.1192484807943959, -1.5885508378195992, 1.971194620192613,
    -5.1806808189868854e-05, 9.39138201062219e-05, 7.466592424760729e-05]
  - [1.3292486467797902, -0.11841761946250584, -1.5869497955215286, 1.989542807847132,
    -5.4355730051116515e-05, 9.064049984050927e-05, 7.555434054442528e-05]
  - [1.3089072257388048, -0.11783311421991888, -1.5858234732570213, 2.0024506318899706,
    -5.6148879080515465e-05, 8.83377415070304e-05, 7.617933530968716e-05]
  - [1.2985658046978197, -0.11753595627790116, -1.5852508597340211, 2.009012869743511,
    -5.706050217041097e-05, 8.716703699065659e-05, 7.649707781169636e-05]
  - [1.2971392657609657, -0.11749496506663497, -1.5851718710260776, 2.0099180923211306,
    -5.718625527806582e-05, 8.700554510578513e-05, 7.654090854339301e-05]
  velocities:
  - [-0.0, 0.0, 0.0, 0.0, -0.0, -0.0, 0.0]
  - [-0.09999999999999908, 0.0028734730056924657, 0.00553708741507194, 0.06345586189298807,
    -8.815259395034306e-06, -1.1320538171050618e-05, 3.072522632527059e-06]
  - [-0.20000000000000007, 0.005746946011384987, 0.011074174830143986, 0.12691172378597737,
    -1.763051879006878e-05, -2.2641076342101452e-05, 6.145045265054176e-06]
  - [-0.30000000000000193, 0.008620419017077533, 0.01661126224521608, 0.1903675856789672,
    -2.6445778185103333e-05, -3.396161451315239e-05, 9.217567897581322e-06]
  - [-0.2999999999999998, 0.008620419017077472, 0.016611262245215964, 0.19036758567896586,
    -2.6445778185103147e-05, -3.396161451315215e-05, 9.217567897581256e-06]
  - [-0.2999999999999998, 0.008620419017077472, 0.016611262245215964, 0.19036758567896586,
    -2.6445778185103147e-05, -3.396161451315215e-05, 9.217567897581256e-06]
  - [-0.29999999999998817, 0.008620419017077137, 0.01661126224521532, 0.19036758567895848,
    -2.644577818510212e-05, -3.396161451315083e-05, 9.217567897580898e-06]
  - [-0.2999999999999921, 0.00862041901707725, 0.016611262245215537, 0.19036758567896098,
    -2.6445778185102466e-05, -3.396161451315128e-05, 9.217567897581018e-06]
  - [-0.299999999999996, 0.008620419017077363, 0.016611262245215752, 0.19036758567896345,
    -2.6445778185102808e-05, -3.396161451315172e-05, 9.217567897581139e-06]
  - [-0.29999999999999594, 0.00862041901707738, 0.01661126224521575, 0.1903675856789634,
    -2.6445778185102804e-05, -3.396161451315171e-05, 9.217567897581118e-06]
  - [-0.2999999999999999, 0.008620419017077493, 0.016611262245215967, 0.1903675856789659,
    -2.644577818510315e-05, -3.396161451315215e-05, 9.217567897581239e-06]
  - [-0.300000000000036, 0.008620419017078532, 0.01661126224521797, 0.19036758567898882,
    -2.6445778185106335e-05, -3.3961614513156244e-05, 9.217567897582348e-06]
  - [-0.30000000000002885, 0.008620419017078326, 0.01661126224521757, 0.1903675856789843,
    -2.6445778185105705e-05, -3.396161451315543e-05, 9.217567897582128e-06]
  - [-0.3000000000000217, 0.00862041901707812, 0.016611262245217175, 0.19036758567897974,
    -2.644577818510507e-05, -3.3961614513154625e-05, 9.217567897581908e-06]
  - [-0.25341421040985984, 0.0072817889287161135, 0.014031766352608405, 0.16080617137488834,
    -2.233911999150738e-05, -2.8687852420314976e-05, 7.786208968882742e-06]
  - [-0.15341421040984504, 0.0044083159230231895, 0.008494678937535595, 0.0973503094818903,
    -1.3523860596471689e-05, -1.736731424926258e-05, 4.713686336355206e-06]
  - [-0.053414210409856944, 0.001534842917331032, 0.0029575915224642623, 0.03389444758890919,
    -4.708601201438349e-06, -6.046776078213203e-06, 1.6411637038284905e-06]
  - [-4.662318002404522e-17, 1.339704492386357e-18, 2.5815662336177665e-18, 2.958514072617761e-17,
    -4.109954257333442e-21, -5.277994891179735e-21, 1.4325077582426342e-21]
  accelerations:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [-0.99999999999932, 0.028734730056905265, 0.05537087415068272, 0.6345586189294635,
    -8.815259395028572e-05, -0.00011320538171043131, 3.072522632525026e-05]
  - [-1.0000000000008882, 0.02873473005695016, 0.05537087415076712, 0.6345586189304462,
    -8.81525939504235e-05, -0.00011320538171060676, 3.072522632529729e-05]
  - [-1.0000000000012768, 0.028734730056961227, 0.05537087415079082, 0.6345586189306933,
    -8.815259395045795e-05, -0.00011320538171065007, 3.0725226325310863e-05]
  - [1.994412619090483e-13, -6.232539434657759e-15, -1.2465078869315518e-14, -9.972063095452415e-14,
    1.2172928583315936e-17, 1.6230571444421247e-17, -6.086464291657968e-18]
  - [2.4917723744801795e-13, -7.786788670250561e-15, -1.5573577340501122e-14, -1.2458861872400898e-13,
    1.5208571621583127e-17, 2.027809549544417e-17, -7.604285810791564e-18]
  - [2.3569186469791314e-11, -6.77752427828506e-13, -1.307093967954976e-12, -1.488289004782597e-11,
    2.06664782879786e-15, 2.6609779233541077e-15, -7.226513649718008e-16]
  - [2.3530099925236306e-11, -6.783545533023671e-13, -1.3049263162763092e-12, -1.483059115006091e-11,
    2.063220552148182e-15, 2.6498224738373714e-15, -7.231385758754659e-16]
  - [2.3743383497796373e-11, -6.801490064472919e-13, -1.3190768609886874e-12, -1.483961468612273e-11,
    2.0731341046422546e-15, 2.656832056434734e-15, -7.245905608458366e-16]
  - [2.366617026058743e-11, -6.78781424426095e-13, -1.3170385847073484e-12, -1.491292920530167e-11,
    2.057872788605232e-15, 2.6514899391644334e-15, -7.222341998470285e-16]
  - [2.4339194149422397e-11, -7.605998171694499e-13, -1.5211996343388998e-12, -1.2169597074711199e-11,
    1.4855465179090819e-15, 2.9710930358181637e-15, -7.427732589545409e-16]
  - [-4.2911375511165716e-11, 1.2335367478893456e-12, 2.3761595646109673e-12, 2.7241119738983472e-11,
    -3.785390067264075e-15, -4.8588588923091115e-15, 1.3176224487864825e-15]
  - [-4.289793716864861e-11, 1.2322844931859396e-12, 2.3717609492281683e-12, 2.7264938911996858e-11,
    -3.786439015411818e-15, -4.853892567628979e-15, 1.3166938863433382e-15]
  - [-4.287565015249122e-11, 1.2336344353338266e-12, 2.3713195256972444e-12, 2.724961397159608e-11,
    -3.781485763691471e-15, -4.859041883964617e-15, 1.315153898345795e-15]
  - [0.9999999999878704, -0.028734730056576833, -0.05537087415004783, -0.6345586189222095,
    8.815259394927559e-05, 0.00011320538170913478, -3.0725226324896095e-05]
  - [1.0000000000126508, -0.02873473005728808, -0.05537087415141919, -0.6345586189378964,
    8.815259395145482e-05, 0.00011320538171193659, -3.0725226325659645e-05]
  - [0.9999999999927625, -0.028734730056717128, -0.055370874150319535, -0.6345586189252893,
    8.81525939497065e-05, 0.00011320538170968858, -3.072522632504828e-05]
  - [1.000000000027597, -0.028734730057717987, -0.055370874152247986, -0.6345586189473985,
    8.815259395277662e-05, 0.00011320538171363136, -3.0725226326118736e-05]
